include/EventInserterPQ.h

```diff
@@ -0,0 +1,44 @@
+#ifndef EVENTINSERTERPQ_H
+#define EVENTINSERTERPQ_H
+
+#include <string>
+#include <vector>
+#include <map>
+#include <libpq-fe.h>
+//#include "Event.h"
+//#include "Root2Event.h"
+//#include "TupleReader.h"
+
```

Comment:
Add a brief description about what your class does. Example:

// class that connects to a postgres database etc.
// example usage: ...
// ...